applications/osmo4_android_studio/app/src/main/java/com/gpac/Osmo4/Osmo4.java

```diff
@@ -389,7 +390,7 @@ private void copyFileOrDir(GpacConfig config, String root, String path) throws I
 		private void copyFile(GpacConfig config, String root, String filename) throws IOException {
 		// if this file exists, do nothing
 
-            if(root == GUI_ROOT_ASSET_DIR){
+            if(root.equals(GUI_ROOT_ASSET_DIR)){
```

Comment:
@RodolpheFouquet please merge if we can, we have external requests relative to this :)